src/user_config.rs

```diff
@@ -53,21 +53,21 @@ impl Default for Theme {
   fn default() -> Self {
     Theme {
       analysis_bar: Color::LightCyan,
-      analysis_bar_text: Color::Black,
+      analysis_bar_text: Color::Reset,
       active: Color::Cyan,
       banner: Color::LightCyan,
       error_border: Color::Red,
       error_text: Color::LightRed,
       hint: Color::Yellow,
       hovered: Color::Magenta,
       inactive: Color::Gray,
-      playbar_background: Color::Black,
+      playbar_background: Color::Reset,
```

Comment:
For dark themes, I think this needs to stay as Black otherwise the the text won't be visible when the track progress overlays the text
# playbar_background: Color::Black,
<img width="158" alt="image" src="https://user-images.githubusercontent.com/12150276/109384483-f3050080-78e4-11eb-8f8a-21c0c4e75180.png">

# playbar_background: Color::Reset,
<img width="167" alt="image" src="https://user-images.githubusercontent.com/12150276/109384499-0d3ede80-78e5-11eb-9a15-de102605e7d4.png">

Reply:
Yeah, I think that can stay black for both dark and light themes (as cyan will probably be pretty close to cyan in most themes and if not, the user can easily fix it themselves with a theme config setting).